src/components/badge/_badge_group.scss

```
@@ -0,0 +1,49 @@
+$gutterTypes: (
```

Please also prefix any variables created 
```suggestion
$euiGutterTypes: (
```